assets/training/finetune_acft_image/components/model_import/mmt/spec.yaml

```diff
@@ -0,0 +1,63 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+type: command
+
+version: 0.0.1
+name: mmtracking_video_multi_object_tracking_model_import
+display_name: Video Multi Object Tracking MMTracking Model Import
+description: Import PyTorch / MLflow model
+
+is_deterministic: True
+
+environment: azureml://registries/azureml/environments/acft-mmtracking-video-gpu/versions/1
+
+code: ../../../src/model_selector
+
+inputs:
+
+  # Model family
+  model_family:
+    type: string
+    optional: true
+    default: MmTrackingVideo
+    enum:
+      - MmTrackingVideo
+    description: Which framework the model belongs to.
+
+  # Model name
+  model_name:
+    type: string
+    optional: true
+    description: Please select models from AzureML Model Assets for all supported models.
+     For MMTracking, provide the model's config name here, same as its specified in MMTracking Model Zoo.
```

Comment:
Extra spaces - same comment for other places in this file.

Reply:
all updated accordingly. no more new lines !! thanks Rupal!